docker-compose-dev.yml

```diff
@@ -87,12 +87,15 @@ services:
       - '3007:3000' # expose ports - HOST:CONTAINER
     environment:
       - NODE_ENV=development
+      - CI=${CI}
```

Comment:
This will also need to be added to the staging/production docker-compose.

Reply:
@apoclyps Thanks! I was wondering why that wasn't running on CI